src/styles/blocks/_container.scss

```diff
@@ -0,0 +1,28 @@
+.container {
+  padding: 50px 40px;
+  box-sizing: border-box;
+  width: 100vw;
+  display: grid;
+  grid-template-columns: repeat(4, $card-space + $card-width);
+  grid-template-rows: repeat(2, auto);
+  justify-content: center;
+  justify-items: center;
+
+  @media screen and (max-width: 1023px) {
+    grid-template-columns: repeat(3, $card-space + $card-width);
+    justify-content: center;
+    justify-items: center;
+  }
+
+  @media screen and (max-width: 767px) {
+    grid-template-columns: repeat(2, $card-space + $card-width);
+    justify-content: center;
+    justify-items: center;
+  }
+
+  @media screen and (max-width: 487px) {
+    grid-template-columns: auto;
+    justify-content: center;
+    justify-items: center;
+  }
```

Comment:
The only thing that is changing is `grid-template-columns`. You can get rid of justify-content and justify items (from the media queries).

Second thing - we typically want to style with mobile first approach. [Check it out](https://zellwk.com/blog/how-to-write-mobile-first-css/)